geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/SynchronizedStripedExecutor.java

```diff
@@ -59,10 +59,13 @@ private Object getSync(Object stripeId) {
 
   private int getStripeIndex(Object stripeId) {
     int hash = stripeId.hashCode();
+
+    hash %= syncs.length;
     if (hash < 0) {
       hash = -hash;
```

Comment:
Apologies if this is obvious and I've forgotten, but is there a reason we don't do `Math.abs(hash)` here?  Also, it's interesting the way `Math.abs()` handles the `INTEGER.MIN_VALUE`:
```
Note that if the argument is equal to the value of Integer.MIN_VALUE, the most negative representable int value, the result is that same value, which is negative.
```

Reply:
😲